client/src/components/footer/footer.js

```diff
@@ -10,14 +12,21 @@ function Footer() {
           <div className="col-lg-12 col-md-12 col-sm-12  col-xs-12">
             <div className="item">Made with ❤️ for the chess community</div>
           </div>
-          <div className="col-lg-6 col-md-6 col-sm-6  col-xs-12">
+          <div className="col-lg-4 col-md-4 col-sm-4  col-xs-12">
             <div className="item">
               <a href="https://www.gnu.org/licenses/gpl-3.0.en.html">
                 GPLv3 License
               </a>
             </div>
           </div>
-          <div className="col-lg-6 col-md-6 col-sm-6  col-xs-12">
+          <div className="col-lg-4 col-md-4 col-sm-4  col-xs-12">
+            <div className="item">
+              <Link to="/contributors">
```

Comment:
The Client Coverage action is failing due to this (an invariant in programming is something you would expect to be always true before a code is executed, here it is `expected` that Link be always used inside a parent Router component)

![image](https://user-images.githubusercontent.com/37269665/111655653-d72fb300-882f-11eb-81e5-d299421fd7f0.png)

@fave77 Should it be changed to an href, or inside a parent `Router` ?

Reply:
I did both of them, but while enclosing Link within the BrowserRouter resulted it to not re-render on click. So whenever I clicked the link the URL would change but the page would not rerender.

Then I tried it with "a" tag and it worked successfully. 
I could try again for router thing and search for some solution if it is preferred. :)